.github/workflows/ci.yml

```diff
@@ -13,7 +13,7 @@ jobs:
         with:
           profile: minimal
           override: true
-          toolchain: 1.58.1
+          toolchain: 1.60.0
```

Comment:
So that PR is only for coverage (the latest version `cargo-tarpaulin`, coverage tool, doesn't compile with the previously used RUST version) 

In general, we set the minimum supported RUST version after reviewing the RUST versions in all the distributions that we support. :)

Reply:
I am no longer seeing any need for the second commit that updates the toolchain version, so I am about to remove it.

Reply:
The cloud CI now says
```
error: package `log v0.4.20` cannot be built because it requires rustc 1.60.0 or newer, while the currently active rustc version is 1.58.1
```
but I can't yet reproduce that locally. I can't now remember for sure whether I saw the problem with `regex-syntax` on my machine or just on the cloud. I still might be able to get this to pass by adjusting the `Cargo.toml`.

Reply:
That (https://github.com/parallaxsecond/rust-psa-crypto/pull/114/commits/b6194e3de9797eb1c722ae3e5f1fdb3e48357734) worked, but it means a restriction on all users of the crate, so would it be better to add a `Cargo.lock`?

Reply:
I think adding `Cargo.lock` is probably best, so I've pushed a new version with that.